src/handlers/systemInstall.ts

```diff
@@ -141,10 +142,17 @@ export default async function systemInstall(
 
   // Update emulsify project config.
   try {
+    // If no checkout was passed along, and the default checkout was used, fetch it
+    // it can be stored in the project config.
+    let checkout = repo.checkout;
+    if (!checkout) {
+      checkout = await getCachedItemCheckout('systems', [repo.name]);
+    }
```

Comment:
We _must_ store a checkout (branch, tag, commit), because we need to make sure everyone working on the project is using the same version of the system.